src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/Querys/ProviderNameSearchProviderQuery.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Nest;
+using Sfa.Das.ApprenticeshipInfoService.Core.Configuration;
+using Sfa.Das.ApprenticeshipInfoService.Core.Models;
+
+namespace Sfa.Das.ApprenticeshipInfoService.Infrastructure.Elasticsearch.Querys
+{
+    public class ProviderNameSearchProviderQuery : IProviderNameSearchProviderQuery
+    {
+        private readonly IElasticsearchCustomClient _elasticsearchCustomClient;
+        private readonly IConfigurationSettings _applicationSettings;
+
+        public ProviderNameSearchProviderQuery(IElasticsearchCustomClient elasticsearchCustomClient, IConfigurationSettings applicationSettings)
+        {
+            _elasticsearchCustomClient = elasticsearchCustomClient;
+            _applicationSettings = applicationSettings;
+        }
+
+        public ISearchResponse<ProviderNameSearchResult> GetResults(string term, int take, int skip)
+        {
+            return _elasticsearchCustomClient.Search<ProviderNameSearchResult>(s => s
+                .Index(_applicationSettings.ProviderIndexAlias)
+                .Type("providerapidocument")
+                .Skip(skip)
+                .Take(take)
+                .Query(q => q
+                    .Bool(b => b
+                        .Filter(IsNotEmployerProvider())
+                        .Must(mu => mu
+                            .QueryString(qs => qs
```

Comment:
The query is taken from das-search isn't it? I'd question the type of query that they wrote. I'm not sure QueryString is the correct one to use. But I guess if it's not causing any problems it's fine, and it's likely to be dropped anyway.